skimage/io/tests/test_pil.py

```diff
@@ -213,7 +213,7 @@ def test_imsave_filelike():
     # save to file-like object
     with expected_warnings(['precision loss',
                             'is a low contrast image']):
-        imsave(s, image)
+        imsave(s, image, check_contrast=True)
```

Comment:
You should add a test where no warning is raised.

Reply:
It looks like the original test only checked that a warning is raised. I am not intending to change the test.